pkg/proxy/ipvs/netlink_linux.go

```diff
@@ -143,52 +148,63 @@ func (h *netlinkHandle) ListBindAddress(devName string) ([]string, error) {
 // 10.0.0.10  scope host  src 10.0.0.10
 // Then cut the unique src IP fields,
 // --> result set: [10.0.0.1, 10.0.0.10]
-
-// If filterDev is specified, the result will discard route of specified device and cut src from other routes.
 func (h *netlinkHandle) GetLocalAddresses(dev, filterDev string) (sets.String, error) {
```

Comment:
Given this is a big refactor for a (very) important function, any thoughts on adding some unit tests for it?

Reply:
Unit test existed but IMHO they mostly test the fake implementation. With the new implementation I have added unit-tests for the `IpIsValidForSet` function.